setup.cfg

```diff
@@ -0,0 +1,2 @@
+[install]
```

Comment:
Nit: Maybe add a comment here linking back to the thread where you learned about this trick?